proto/regen/ecocredit/basket/v1/tx.proto

```diff
@@ -22,7 +22,7 @@ service Msg {
   rpc Take(MsgTake) returns (MsgTakeResponse);
 }
 
-// MsgCreateBasket is the Msg/CreateBasket request type.
+// MsgCreate is the Msg/Create request type.
```

Comment:
proto changes  should go to master first. hmm... if all changes will be about comments then we can push here, and later copy proto to master?

Reply:
both @amaurym and have been changing only comments in proto files as part of our audit PRs https://github.com/regen-network/regen-ledger/pull/781 and https://github.com/regen-network/regen-ledger/pull/780
so we could handle all these changes could be handled the way you suggest

Reply:
ok, we will do manual diff